db/migrations/versions/seed_event_codes.py

```diff
@@ -0,0 +1,50 @@
+"""empty message
+
+Revision ID: seed_event_codes
+Revises: initial_migration
+Create Date: 2022-07-15 13:24:40.0
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.sql import table
+
+# revision identifiers, used by Alembic.
+revision = "seed_event_codes"
+down_revision = "initial_migration"
+branch_labels = None
+depends_on = None
+codes = [
+    {
+        "id": "SCORE_CREATED",
+        "description": "A score was created in the assessment frontend",
+    },
+    {
+        "id": "SCORE_AMENDED",
+        "description": "A score was amended in the assessment frontend",
+    },
+    {
+        "id": "SCORE_SUBMITTED",
+        "description": "A score was submitted in the assessment frontend",
+    },
+]
+
+
+def upgrade():
+    Code = table(
+        "Code",
+        sa.Column("id", sa.String(length=5), nullable=False),
+        sa.Column("description", sa.Text(), nullable=True),
+    )
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.bulk_insert(Code, codes)
```

Comment:
just curious! how do you get flask to generate this migration as its populating the db with some data?

Reply:
I wrote this migration ;)

Reply:
ahhh amazing! will have to remember this trick to create some dummy data :D